css/events.css

```diff
@@ -10,6 +10,9 @@ header {
 	min-height: auto;
 }
 
+section {
+	padding: var(--spacing-lg) 0;
+}
```

Comment:
applying on events and event-single pages

Reply:
Looks better, thanks